app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -590,11 +682,25 @@ $search-icon-size: 20px;
       margin-bottom: 0;
       padding-left: govuk-spacing(3);
       padding-right: govuk-spacing(3);
-      width: 50%;
     }
   }
 }
 
+.gem-c-layout-super-navigation-header__navigation-second-items--topics {
+  @include govuk-media-query($from: "desktop") {
+    // columns(number-of-items, number-of-columns)
+    @include columns(18, 2, "li");
+  }
+}
+
+.gem-c-layout-super-navigation-header__navigation-second-items--government-activity {
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(3);
+
```

Comment:
Do we want this space here?

Bonus question: do we reckon it's overkill to pop the comment from 691 here?

Reply:
Ah, I meant to remove the comment from 691 as well because there are examples and some documentation above the mixin itself:

https://github.com/alphagov/govuk_publishing_components/blob/9ce1fecf803a9577a336c0aa4fdadb30410558b9/app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss#L22-L38

I think best to keep examples in one place - less places to change to do if anything needs updating.

Reply:
Fine by me 👍

Reply:
And yes, we want the space - the grid makes the spacing a bit different for each dropdown navigation:

Live:

![image](https://user-images.githubusercontent.com/1732331/133305995-cd45748e-8a61-4932-9f9d-cd33e7460c8c.png)


Without spacing adjustment:

![image](https://user-images.githubusercontent.com/1732331/133306093-cdf461a7-00fb-4a40-a783-0b6ad2659cc2.png)


With spacing adjustment:
![image](https://user-images.githubusercontent.com/1732331/133306024-d44fedb9-f109-4887-8cd6-1c7db2a102aa.png)

Reply:
Just to clarify that we're talking about the same thing, cause that's bananas if this is the case: I'm referring to the blank line on 699 and proposing that it's unnecessary. Is that extra line changing the spacing behaviour of the component?

Reply:
uh, bother - I thought you meant the `govuk-spacing(3)`. The unnecessary line is indeed unnecessary, and has been removed. 🐑

Reply:
Mystery solved 😂 

I've already approved this so happy for you to hit merge.